docs/whats_new/v0.4.0.md

```diff
@@ -0,0 +1,152 @@
+# Version 0.4.0
+
+v0.4.0 contains major enhancements to Tabular and Text modules, along with many quality of life improvements and fixes.
+
+TODO: Finalize high level description, add links to documentation, add Text module updates, add links to PRs
+
+This release contains **149** commits from **14** contributors!
+
+Special thanks to @zhiqiangdon, @willsmithorg, @DolanTheMFWizard, @truebluejason, @killerSwitch, and @Xilorole who were first time contributors to AutoGluon this release!
+
+Full Contributor List (ordered by # of commits):
+
+@Innixma, @yinweisu, @gradientsky, @zhiqiangdon, @jwmueller, @willsmithorg, @sxjscience, @DolanTheMFWizard, @truebluejason, @taesup-aws, @Xilorole, @mseeger, @killerSwitch, @rschmucker
+
+# Changes
+
+## General
+
+- AutoGluon now supports Windows OS! Both CPU and GPU are supported on Windows. @Innixma, @yinweisu (#1463, #1470)
+- AutoGluon now supports Python 3.9! @yinweisu (#1330)
+- Python 3.6 support has been dropped.
```

Comment:
Consider to merge the python 3.9 support / python 3.6 drop.

Reply:
Merged